grakn-graql/src/main/java/ai/grakn/graql/internal/template/Value.java

```diff
@@ -111,14 +113,7 @@ public static Value concat(Value... values){
 
     // FORMATS
 
-    public static Function<Value, String> format = (val) -> {
-        if(val.isString()){
-            return "\"" + val + "\"";
-        }
-
-        return val.toString();
-    };
-
+    public static Function<Value, String> format = (val) -> StringConverter.valueToString(val.value);
```

Comment:
Instead of `static Function<Value, String>`, these might as well be _actual_ static functions:
`static String format(Value val) { ... }`